lib/model/JournalEntry.php

```diff
@@ -151,6 +155,18 @@ public function getLink($oPage = null, $sSubpage = null) {
 		return $oPage->getLinkArray($this->getPublishAt('Y'), $this->getPublishAt('n'), $this->getPublishAt('j'), $this->getSlug());
 	}
 
+	public function getNextJournalEntry() {
+		return self::getNeighbourQuery($this->getJournalId(), $this->getId())->filterByPublishAt($this->getPublishAt('Y-m-d'), Criteria::GREATER_THAN)->orderByPublishAt(Criteria::ASC)->findOne();
+	}
+
+	public function getPreviousJournalEntry() {
+		return self::getNeighbourQuery($this->getJournalId(), $this->getId())->filterByPublishAt($this->getPublishAt('Y-m-d'), Criteria::LESS_THAN)->orderByPublishAt(Criteria::DESC)->findOne();
+	}
+
+	private static function getNeighbourQuery($iJournalId, $iCurrendId) {
```

Comment:
I changed this with commit 8e7c98f334d0da3b92d04f809d2cb3c285204c40
and added a comment, since now the method is only returning a sibling, but not neighbour. See comment in commit